packages/lib-classifier/src/plugins/tasks/survey/components/components/Choice/Choice.js

```diff
@@ -23,22 +31,39 @@ export default function Choice({
     questions,
     strings
   } = task
+  const choiceRef = useRef(null)
+
+  useEffect(() => {
+    if (choiceRef.current) {
+      choiceRef.current.focus()
+    }
+  }, [choiceId])
 
   const { t } = useTranslation('plugins')
 
   const choice = choices?.[choiceId] || {}
   const questionIds = getQuestionIds(choiceId, task)
   const allowIdentify = allowIdentification(answers, choiceId, task)
 
+  function handleKeyDown (event) {
+    if (event.key === 'Escape') {
+      handleDelete(choiceId)
+    }
+  }
+
   return (
-    <Box
+    <StyledBox
+      ref={choiceRef}
+      aria-labelledby='choice-label'
```

Comment:
```suggestion
      aria-labelledby='choice-label'
      as=‘section’
```
I’d recommend using `section` here as `div` can’t be labelled.